### PR TITLE
added outlook to the list of protected products 

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#920

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/set-up-atp-safe-links-policies.md [hunk 1]
```diff
@@ -117,8 +117,8 @@ Default policy options apply to everyone in your organization.
...
-| **Office 365 ProPlus, Office for iOS and Android** <br/>    | When this option is selected, ATP Safe Links protection is applied to URLs in Word, Excel, and PowerPoint files on Windows or Mac OS, Office documents on iOS, or Android devices, Visio 2016 on Windows, and the web versions of Office apps (Word, PowerPoint, Excel, and OneNote), provided the user has signed into Office 365. |
-| **Don't track when users click ATP Safe Links** <br/>  | When this option is selected, click data for URLs in Word, Excel, PowerPoint, and Visio documents is not stored.  <br/> |
+| **Office 365 ProPlus, Office for iOS and Android** <br/>    | When this option is selected, ATP Safe Links protection is applied to URLs in Word, Excel, and PowerPoint files on Windows or Mac OS, emails in Outlook, Office documents on iOS, or Android devices, Visio 2016 on Windows, and the web versions of Office apps (Word, PowerPoint, Excel, Outlook and OneNote), provided the user has signed into Office 365. |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-atp-safe-links-policies.md]:
```suggestion
| **Office 365 ProPlus, Office for iOS and Android** <br/>    | When this option is selected, ATP Safe Links protection is applied to URLs in Word, Excel, and PowerPoint files on Windows or Mac OS, emails in Outlook, Office documents on iOS or Android devices, Visio 2016 on Windows, and the web versions of Office apps (Word, PowerPoint, Excel, Outlook, and OneNote), provided the user has signed in to Office 365. |
```

File: microsoft-365/security/office-365-security/set-up-atp-safe-links-policies.md [hunk 2]
```diff
@@ -117,8 +117,8 @@ Default policy options apply to everyone in your organization.
...
+| **Don't track when users click ATP Safe Links** <br/>  | When this option is selected, click data for URLs in Word, Excel, PowerPoint, and Visio documents and Outlook emails is not stored.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-atp-safe-links-policies.md]:
```suggestion
| **Don't track when users click ATP Safe Links** <br/>  | When this option is selected, click data for URLs in Word, Excel, PowerPoint, Visio documents, and Outlook emails is not stored.  <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #920

Thank you.

cc: @kenwith

**Review by denisebmsft — APPROVED** [review]

**Pull request merged**